README.md

```diff
@@ -5,10 +5,16 @@
 
 После успешной настройки переменных окружения, нужно перейти в папку expenses_app и открыть из нее консоль Git-Bash
 
-В консоли выполнить последовательно команды:
+## Для запуска приложения с помощью docker-compose
 
-Для компиляции приложения: mvn compile
+### Выполнить в консоли команды:
 
-Для запуска приложения: mvn exec:java -Dexec.mainClass="com.example.vikhlia.ExpensesAppApplication"
+mvn clean
+
+mvn package
```

Comment:
Я все еще получаю ошибку при выполнении `mvn package`

```
➜  expenses_app git:(feature/1-setup-docker-and-postgresql) mvn package
[INFO] Scanning for projects...
[INFO]
[INFO] ------------------------< com.example:vikhlia >-------------------------
[INFO] Building expenses_app 1.0.1-SNAPSHOT
[INFO]   from pom.xml
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- resources:3.3.1:resources (default-resources) @ vikhlia ---
[INFO] Copying 1 resource from src/main/resources to target/classes
[INFO] Copying 3 resources from src/main/resources to target/classes
[INFO]
[INFO] --- compiler:3.13.0:compile (default-compile) @ vikhlia ---
[INFO] Recompiling the module because of changed source code.
[INFO] Compiling 13 source files with javac [debug parameters release 17] to target/classes
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /Users/madrazzl3/projects/mentoring/expenses_app/src/main/java/com/example/vikhlia/controller/StartPageController.java:[49,62] cannot find symbol
  symbol:   method getCategory()
  location: variable viewTransaction of type com.example.vikhlia.utils.ViewTransaction
[ERROR] /Users/madrazzl3/projects/mentoring/expenses_app/src/main/java/com/example/vikhlia/controller/StartPageController.java:[63,25] cannot find symbol
  symbol:   method setName(java.lang.String)
  location: variable category of type com.example.vikhlia.entity.Category
[ERROR] /Users/madrazzl3/projects/mentoring/expenses_app/src/main/java/com/example/vikhlia/utils/UtilsService.java:[22,28] cannot find symbol
  symbol:   method getName()
  location: variable category of type com.example.vikhlia.entity.Category
[ERROR] /Users/madrazzl3/projects/mentoring/expenses_app/src/main/java/com/example/vikhlia/utils/UtilsService.java:[32,40] cannot find symbol
  symbol:   method getAmount()
  location: variable transaction of type com.example.vikhlia.entity.Transaction
[ERROR] /Users/madrazzl3/projects/mentoring/expenses_app/src/main/java/com/example/vikhlia/utils/UtilsService.java:[41,39] cannot find symbol
  symbol:   method getAmount()
  location: variable transaction of type com.example.vikhlia.entity.Transaction
[ERROR] /Users/madrazzl3/projects/mentoring/expenses_app/src/main/java/com/example/vikhlia/utils/UtilsService.java:[47,28] cannot find symbol
  symbol:   method getType()
  location: variable transaction of type com.example.vikhlia.entity.Transaction
[ERROR] /Users/madrazzl3/projects/mentoring/expenses_app/src/main/java/com/example/vikhlia/utils/UtilsService.java:[55,42] cannot find symbol
  symbol:   method getId()
  location: variable viewTransaction of type com.example.vikhlia.utils.ViewTransaction
[ERROR] /Users/madrazzl3/projects/mentoring/expenses_app/src/main/java/com/example/vikhlia/utils/UtilsService.java:[56,46] cannot find symbol
  symbol:   method getAmount()
  location: variable viewTransaction of type com.example.vikhlia.utils.ViewTransaction
[ERROR] /Users/madrazzl3/projects/mentoring/expenses_app/src/main/java/com/example/vikhlia/utils/UtilsService.java:[57,20] cannot find symbol
  symbol:   method setDate(java.lang.String)
  location: variable transaction of type com.example.vikhlia.entity.Transaction
[ERROR] /Users/madrazzl3/projects/mentoring/expenses_app/src/main/java/com/example/vikhlia/utils/UtilsService.java:[58,63] cannot find symbol
  symbol:   method getType()
  location: variable viewTransaction of type com.example.vikhlia.utils.ViewTransaction
[ERROR] /Users/madrazzl3/projects/mentoring/expenses_app/src/main/java/com/example/vikhlia/utils/UtilsService.java:[66,25] cannot find symbol
  symbol:   method getName()
  location: variable category of type com.example.vikhlia.entity.Category
[ERROR] /Users/madrazzl3/projects/mentoring/expenses_app/src/main/java/com/example/vikhlia/utils/UtilsService.java:[67,30] cannot find symbol
  symbol:   method getId()
  location: variable category of type com.example.vikhlia.entity.Category
[INFO] 12 errors
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.805 s
[INFO] Finished at: 2024-12-24T13:26:08+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.13.0:compile (default-compile) on project vikhlia: Compilation failure: Compilation failure:
[ERROR] /Users/madrazzl3/projects/mentoring/expenses_app/src/main/java/com/example/vikhlia/controller/StartPageController.java:[49,62] cannot find symbol
[ERROR]   symbol:   method getCategory()
[ERROR]   location: variable viewTransaction of type com.example.vikhlia.utils.ViewTransaction
[ERROR] /Users/madrazzl3/projects/mentoring/expenses_app/src/main/java/com/example/vikhlia/controller/StartPageController.java:[63,25] cannot find symbol
[ERROR]   symbol:   method setName(java.lang.String)
[ERROR]   location: variable category of type com.example.vikhlia.entity.Category
[ERROR] /Users/madrazzl3/projects/mentoring/expenses_app/src/main/java/com/example/vikhlia/utils/UtilsService.java:[22,28] cannot find symbol
[ERROR]   symbol:   method getName()
[ERROR]   location: variable category of type com.example.vikhlia.entity.Category
[ERROR] /Users/madrazzl3/projects/mentoring/expenses_app/src/main/java/com/example/vikhlia/utils/UtilsService.java:[32,40] cannot find symbol
[ERROR]   symbol:   method getAmount()
[ERROR]   location: variable transaction of type com.example.vikhlia.entity.Transaction
[ERROR] /Users/madrazzl3/projects/mentoring/expenses_app/src/main/java/com/example/vikhlia/utils/UtilsService.java:[41,39] cannot find symbol
[ERROR]   symbol:   method getAmount()
[ERROR]   location: variable transaction of type com.example.vikhlia.entity.Transaction
[ERROR] /Users/madrazzl3/projects/mentoring/expenses_app/src/main/java/com/example/vikhlia/utils/UtilsService.java:[47,28] cannot find symbol
[ERROR]   symbol:   method getType()
[ERROR]   location: variable transaction of type com.example.vikhlia.entity.Transaction
[ERROR] /Users/madrazzl3/projects/mentoring/expenses_app/src/main/java/com/example/vikhlia/utils/UtilsService.java:[55,42] cannot find symbol
[ERROR]   symbol:   method getId()
[ERROR]   location: variable viewTransaction of type com.example.vikhlia.utils.ViewTransaction
[ERROR] /Users/madrazzl3/projects/mentoring/expenses_app/src/main/java/com/example/vikhlia/utils/UtilsService.java:[56,46] cannot find symbol
[ERROR]   symbol:   method getAmount()
[ERROR]   location: variable viewTransaction of type com.example.vikhlia.utils.ViewTransaction
[ERROR] /Users/madrazzl3/projects/mentoring/expenses_app/src/main/java/com/example/vikhlia/utils/UtilsService.java:[57,20] cannot find symbol
[ERROR]   symbol:   method setDate(java.lang.String)
[ERROR]   location: variable transaction of type com.example.vikhlia.entity.Transaction
[ERROR] /Users/madrazzl3/projects/mentoring/expenses_app/src/main/java/com/example/vikhlia/utils/UtilsService.java:[58,63] cannot find symbol
[ERROR]   symbol:   method getType()
[ERROR]   location: variable viewTransaction of type com.example.vikhlia.utils.ViewTransaction
[ERROR] /Users/madrazzl3/projects/mentoring/expenses_app/src/main/java/com/example/vikhlia/utils/UtilsService.java:[66,25] cannot find symbol
[ERROR]   symbol:   method getName()
[ERROR]   location: variable category of type com.example.vikhlia.entity.Category
[ERROR] /Users/madrazzl3/projects/mentoring/expenses_app/src/main/java/com/example/vikhlia/utils/UtilsService.java:[67,30] cannot find symbol
[ERROR]   symbol:   method getId()
[ERROR]   location: variable category of type com.example.vikhlia.entity.Category
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```